app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileResetPinActivityTest.kt

```diff
@@ -120,6 +131,30 @@ class ProfileResetPinActivityTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
+  @Test
+  fun testProfileResetPinActivity_hasCorrectActivityLabel() {
+    activityTestRule.launchActivity(
+      createProfileResetPinActivityIntent(
+        internalProfileId,
```

Comment:
Just one nit. Add named argument.

`internalProfileId = internalProfileId,`

Reply:
This actually shouldn't be needed, I think. Named arguments generally provide benefit when contextualizing literals, but in this case 'internalProfileId' is already giving us all the context that's needed.